common/data_refinery_common/migrations/0038_paginate_default_svd_algorithm.py

```diff
@@ -0,0 +1,40 @@
+from django.db import migrations, models
+from django.core.paginator import Paginator
+
+def set_computed_file_svd_algorithm_none(apps, schema_editor):
+    """ Set svd_algorithm to NONE for existing computed files that are not compendia"""    
+    ### is_compendia=False
+    ComputedFile = apps.get_model('data_refinery_common', 'ComputedFile')
+    paginator = Paginator(ComputedFile.objects.all().order_by('id').filter(is_compendia=False), 1000)
+    for page_idx in range(1, paginator.num_pages):
+        for computedFile in paginator.page(page_idx).object_list:
+            computedFile.update(svd_algorithm="NONE")
+
+def set_computed_file_svd_algorithm_arpack(apps, schema_editor):
+    """ Set svd_algorithm to ARPACK for exiting compendia """
+    ### is_compendia=True   
+    ComputedFile = apps.get_model('data_refinery_common', 'ComputedFile')
+    paginator = Paginator(ComputedFile.objects.all().order_by('id').filter(is_compendia=True), 1000)
+    for page_idx in range(1, paginator.num_pages):
+        for computedFile in paginator.page(page_idx).object_list:
+            computedFile.update(svd_algorithm="ARPACK")
```

Comment:
Can this variable name be computed_file instead of computedFile?

Reply:
OYE haha too much javascript 👍